setup.py

```diff
@@ -500,7 +507,7 @@ def copy_recursively(source_folder, destination_folder):
         'src',
         exclude=['ez_setup', 'setup', 'examples', 'tests']),
     namespace_packages=["leap"],
-    package_data={'': ['util/*.txt']},
+    package_data={'': ['util/*.txt', '*.pem']},
```

Comment:
Which pem files do we need?
All I see is tests related data and it seems that we exclude tests.

```
[bitmask_client/] ➜ find -iname "*.pem"
./src/leap/bitmask/services/eip/tests/wrongcert.pem
./src/leap/bitmask/crypto/tests/wrongcert.pem
./src/leap/bitmask/crypto/tests/openvpn.pem
```